### ISSUE TITLE
Screensaver on linux

### ISSUE THREAD

**Issue by mjrev** [Giphy/GiphyAPI#48]
hey. yesterday i managed to set one of giphy tv page as screensaver on my ~~[#!](http://crunchbang.org/)~~ debian-based linux, it looks pretty awesome - http://tv.giphy.com/?username=mrdiv
here's instruction:
- `sudo apt-get install xscreensaver xscreensaver-gl xscreensaver-data xscreensaver-data-extra python2.7 python2.7-dev python-gtk2 python-gtk2-dev libxslt1-dev libwebkit-dev python-webkit`
- install pywebkitgtk https://code.google.com/p/pywebkitgtk/
- install webscreensaver https://github.com/lmartinking/webscreensaver
- add `xscreensaver -no-splash &` to autostart script
- run `xscreensaver-demo` to configure screensaver

it will be cool if some sort of instruction or link to this page will be on [screensavers page](http://giphy.com/screensavers) so other people could do the same.
basically, it allows to set any webpage as screensaver.

**Comment by dfinkler** [comment]
:+1: 

Nice one. Thanks for the note @mjrev - we'll be sure to include your methodology when we update the screensaver page.
